README.md

```diff
@@ -15,6 +15,297 @@ This repository contains two smart contracts:
 - [**Cert Wallet Contract**](https://github.com/kommitters/chaincerts_governance/tree/main/certs_wallet):
   The `cert_wallet` contract stores the Chaincerts of a specific user, and each user must have their own `cert_wallet`. This contract allows users to view their Chaincerts, add organizations authorized to issue certificates to them, and execute distribution and revocation actions on their certificates. Only functions that can be executed by the `cert_governance` contract can modify certificates in a `cert_wallet`.
 
+## System overview
+The following image shows the communication of each contract with the Stellar network and the interaction between them.
+![Flow of the system](how_it_works.png)
+
+## Pre-requirements
+To be able to work with the contracts, you first need some programs or dependencies, for this you can read the following guide: [Soroban setup](https://soroban.stellar.org/docs/getting-started/setup).
+
+## Setup
+``` 
+# Clone the repository 
+git clone git@github.com:kommitters/chaincerts_governance.git
+
+# Build the project and get dependencies 
+cd chaincerts_governance
+cargo build
+ ```
+
+## Testing
+To test the contract run `cargo test -- --show-output` or ` cargo test -- --nocapture`
+
+## Contracts workflow
```

Comment:
In general, I think there are things that are unclear, and if you don't have enough context you're going to get lost with these steps, let's discuss a better approach for explaining the contracts workflow, which can also be more concrete